lib/smart_answer_flows/benefit-cap-calculator.rb

```diff
@@ -175,5 +175,8 @@ def define
       ## Outcome 5
       outcome :outcome_not_affected
     end
+    def self.next_benefit_amount_question(benefits, selected_benefits)
+      benefits.fetch(selected_benefits.shift, :housing_benefit_amount?)
+    end
```

Comment:
benefits.fetch(selected_benefits.shift, :housing_benefit_amount?) was moved to a function to make the benefit cap flow readable.